src/commands/initProjectForVSCode/InitVSCodeStep/PythonInitVSCodeStep.ts

```diff
@@ -7,7 +7,7 @@ import * as fse from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
 import { DebugConfiguration, TaskDefinition } from 'vscode';
-import { extensionPrefix, extInstallCommand, func, funcWatchProblemMatcher, gitignoreFileName, hostStartCommand, packTaskName, Platform, pythonVenvSetting } from "../../../constants";
+import { extensionPrefix, extInstallCommand, extInstallTaskName, func, funcWatchProblemMatcher, gitignoreFileName, hostStartCommand, packTaskName, Platform, pythonVenvSetting } from "../../../constants";
```

Comment:
`extInstallCommand` isn't used anymore

Reply:
![are you sure](https://media1.tenor.com/images/fb1b9f4775f881f7784815573b5361b1/tenor.gif?itemid=8602294)

Reply:
Stinking expand code button....